packages/config/src/common/ProjectProvider.ts

```diff
@@ -0,0 +1,11 @@
+export type ProjectProvider =
```

Comment:
Maybe lets create L3ProjectProvider or smth like this to not mix the L2 providers with L3?